inst/js/shiny-tracer.js

```diff
@@ -296,6 +296,21 @@ window.shinytest = (function() {
 
     // This sets shinytest.ready to true when the Shiny application is ready.
     function waitForReady() {
+        // It's possible (though unusual) to get here before jQuery is
+        // loaded. In that case, wait until jQuery is loaded before doing the
+        // rest of the stuff in this function.
+        var jquery_message_shown = false;
```

Comment:
```suggestion
    var jquery_message_shown = false;
    // This sets shinytest.ready to true when the Shiny application is ready.
    function waitForReady() {
        // It's possible (though unusual) to get here before jQuery is
        // loaded. In that case, wait until jQuery is loaded before doing the
        // rest of the stuff in this function.
```

Reply:
This would have the message only show once.

Otherwise, I would remove the variable as it is reset each time